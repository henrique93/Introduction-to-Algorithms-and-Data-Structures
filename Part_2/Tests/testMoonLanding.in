5
send @Tintin #MoonLanding L'instant est solennel... La porte extérieure tourne lentement sur ces gonds et...
send @Tintin #MoonLanding Ooooh!... Quel spectacle hallucinant!
send @Tintin #MoonLanding Je descends à présent les échelons qui courent le long de la fusée...
send @Tintin #MoonLanding Ça y est!... J'ai fait quelques pas!...
send @Tintin #MoonLanding Pour la première fois sans doute dans l'histoire de l'humanité, ON A MARCHÉ SUR LA LUNE!
send @Armstrong #MoonLanding Okay. I'm going to step off the LM now.
close @Tintin #MoonLanding
send @Armstrong #MoonLanding That's one small step for man; one giant leap for mankind.
send @Armstrong #MoonLanding Yes, the surface is fine and powdery. I can kick it up loosely with my toe.
send @Armstrong #MoonLanding Ah... There seems to be no difficulty in moving around - as we suspected.
close @Tintin #MoonLanding
send @Wallace #Cheese Gromit, that's it! Cheese! We'll go somewhere where there's cheese!
send @Wallace #Cheese Everybody knows the moon's made of cheese.
list #Gorgonzola
list #MoonLanding
close @Armstrong #MoonLanding
send @Philae2014 #CometLanding Thank you, @ESA_Rosetta! I did it! I became the first spacecraft to land on a comet & study it! But it's not over yet...
send @ESA_Rosetta #CometLanding You've done a great job Philae, something no spacecraft has ever done before.
send  @Philae2014 #CometLanding I'm feeling a bit tired, did you get all my data? I might take a nap...
list #MoonLanding
list #CometLanding
send @Wallace #MoonLanding Everything seems to be under contro-o-ol!
send @Wallace #MoonLanding Ooh... seconds to blast-off.
send @Wallace #MoonLanding No crackers, Gromit! We've forgotten the crackers!
send @Wallace #MoonLanding Hold on, Gromit! Hold on!
send @Wallace #MoonLanding Adjust angle of thrust. Steady now. Easing up.
send @Wallace #MoonLanding Ah! Plate. Knife. Cracker.
send @Wallace #MoonLanding I don't know, lad. It's like no cheese I've ever tasted. Let's try another spot.
list #Cheese
list #MoonLanding
quit