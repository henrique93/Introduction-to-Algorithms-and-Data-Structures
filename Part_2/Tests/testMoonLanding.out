NULL
@Armstrong #MoonLanding : That's one small step for man; one giant leap for mankind.
@Armstrong #MoonLanding : Yes, the surface is fine and powdery. I can kick it up loosely with my toe.
@Armstrong #MoonLanding : Ah... There seems to be no difficulty in moving around - as we suspected.
NULL
@Philae2014 #CometLanding : Thank you, @ESA_Rosetta! I did it! I became the first spacecraft to land on a comet & study it! But it's not over yet...
@ESA_Rosetta #CometLanding : You've done a great job Philae, something no spacecraft has ever done before.
@Philae2014 #CometLanding : I'm feeling a bit tired, did you get all my data? I might take a nap...
@Wallace #Cheese : Gromit, that's it! Cheese! We'll go somewhere where there's cheese!
@Wallace #Cheese : Everybody knows the moon's made of cheese.
@Wallace #MoonLanding : Everything seems to be under contro-o-ol!
@Wallace #MoonLanding : Ooh... seconds to blast-off.
@Wallace #MoonLanding : No crackers, Gromit! We've forgotten the crackers!
@Wallace #MoonLanding : Hold on, Gromit! Hold on!
@Wallace #MoonLanding : Adjust angle of thrust. Steady now. Easing up.
@Wallace #MoonLanding : Ah! Plate. Knife. Cracker.
@Wallace #MoonLanding : I don't know, lad. It's like no cheese I've ever tasted. Let's try another spot.
